src/plays/zoomlogin/components/App.css

```diff
@@ -0,0 +1,52 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
please change the class name here too

Reply:
I removed this file as it's not used in the project.